pygmt/base_plotting.py

```diff
@@ -115,6 +115,9 @@ def coast(self, **kwargs):
             ``'river[/pen]'``
             Draw rivers. Specify the type of rivers and [optionally] append pen
             attributes.
+        L : str
```

Comment:
Do you mind adding an alias for `L` to `coast`? [GMT.jl](https://www.generic-mapping-tools.org/GMT.jl/latest/#GMT.coast) uses `map_scale` and I think we can copy that. It'll just be a one line addition.

Reply:
Ok, I think that's done in the new commit. Thanks!